operators/src/pro/meta/wrapper.rs

```diff
@@ -137,6 +152,10 @@ where
         query: QueryRectangle<Self::SpatialBounds>,
         ctx: &'a dyn QueryContext,
     ) -> Result<BoxStream<'a, Result<Self::Output>>> {
+        let qc = self.inc_query_count() + 1;
+
+        tracing::debug!(event = "query", query_count = qc, path = ?self.path);
```

Comment:
should maybe be trace level? Also include the query rectangle in the output?

Reply:
https://github.com/geo-engine/geoengine/pull/780/commits/f088e93e3762d8e0ca51deafcffd71336be94aad
https://github.com/geo-engine/geoengine/pull/780/commits/0a56dd8de4353917ddf5b77fb1eff2060b6962e4
changed the display method again here : https://github.com/geo-engine/geoengine/pull/780/commits/b286ed0c8e8a84e6f3111f7f9605a1184529f5fd